projects/frontegg-app/src/lib/frontegg-entitlements.service.ts

```diff
@@ -1,16 +1,13 @@
 import { Subscription, PartialObserver, BehaviorSubject } from 'rxjs';
 import { AuthState } from '@frontegg/redux-store';
 import { Injectable } from '@angular/core';
-import { Entitlement, LoadEntitlementsCallback, EntitledToOptions } from '@frontegg/types';
+import { Entitlement, LoadEntitlementsCallback, EntitledToOptions, CustomAttributes } from '@frontegg/types';
 
-import {
-  FronteggState,
-  getPermissionEntitlements,
-  getFeatureEntitlements,
-  getEntitlements,
-} from '@frontegg/redux-store';
+import { FronteggState } from '@frontegg/redux-store';
 import { FronteggAppService } from './frontegg-app.service';
 
+type User = FronteggState['auth']['user'];
```

Comment:
dont we already have this type in admin-box?

Reply:
good point, we have `User` but I need this one because it's `User | undefined | null`